raster/r.patch/main.c

```diff
@@ -154,14 +169,18 @@ int main(int argc, char *argv[])
             out_type = DCELL_TYPE;
 
         Rast_init_cell_stats(&statf[i]);
```

Comment:
I would call `Rast_init_cell_stats` in a loop here instead of memcpy.

Reply:
Right. I think there are more ways to proceed, but first you need to make sure you understand how support() works and how it's "merging" the stats from multiple rasters. It's using the stats from the first raster as a base (see the function documentation).  So then one way it could work is that you first "merge" the stats for the first raster (with `Rast_find_cell_stat` and `Rast_update_cell_stat`) and then continue the loop in support() but looping through the rest of your stats (adding a loop over threads). Or you can merge them all before calling support() and then you don't need to modify anything there. I don't know what is better performance-wise.